docs/source/reference/websecurity.md

```diff
@@ -1,24 +1,21 @@
 # Security Overview
 
-The **Security Overview** section helps you learn about:
-
-- the design of JupyterHub with respect to web security
-- the semi-trusted user
-- the available mitigations to protect untrusted users from each other
-- the value of periodic security audits
+The **Security Overview** section helps you learn about: the semi-trusted user;
+the available mitigations to protect untrusted users from each other;
+the value of periodic security audits.
 
 This overview also helps you obtain a deeper understanding of how JupyterHub
-works.
+works, as well as its design with respect to web security.
 
 ## Semi-trusted and untrusted users
 
 JupyterHub is designed to be a _simple multi-user server for modestly sized
-groups_ of **semi-trusted** users. While the design reflects serving semi-trusted
-users, JupyterHub is not necessarily unsuitable for serving **untrusted** users.
+groups_ of **semi-trusted** users. Its design reflects serving semi-trusted
+users, however, JupyterHub is not necessarily unsuitable for serving **untrusted** users.
```

Comment:
I prefer the `While...` construction here. Can you discuss why you think the `however` works better?

Reply:
The phrase sounds confusing to me, so I added the `however` so it could bring an idea of opposition between semi-trusted and untrusted user's cases.

Reading it again and again, maybe my discomfort is on `is not necessarily unsuitable`. It could be, then, `While the design reflects serving semi-trusted users, JupyterHub can also be suitable for serving **untrusted** users.`?